force-app/main/default/classes/SOQL.cls

```diff
@@ -345,17 +416,31 @@ public with sharing class SOQL {
      */
     public static void setPrimaryContact() {
         //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
+        List<Opportunity> oppsClosedWon = getOpportunityDetailsForClosedWon();
 
         // Create a set of Account Ids from the Opportunities
+        Set<Id> accIds = new Set<Id>();
+        for (Opportunity opp : oppsClosedWon) {
+            accIds.add(opp.AccountId);
+        }
 
         // Query the Contacts
+        List<Contact> conts = [SELECT Id, AccountId FROM Contact WHERE AccountId = :accIds];
```

Comment:
Is there a way to use a subquery to get the account and contacts at the same time?